packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -487,25 +487,29 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
         // If there is no legend, need not to allocate some space from total chart space.
         legendContainerHeight = 0;
       } else {
-        const legendContainerComputedStyles = getComputedStyle(this.legendContainer);
+        const legendContainerComputedStyles = this.legendContainer && getComputedStyle(this.legendContainer);
```

Comment:
why is this change needed

Reply:
I got the following error when I used wait after mount:
![WhatsApp Image 2022-08-08 at 1 47 35 PM](https://user-images.githubusercontent.com/110246001/185084797-3f4cd300-c2ca-421c-87b8-2deb08166d45.jpeg)